extensions/wikia/WikiaMaps/controllers/WikiaMapsPoiController.class.php

```diff
@@ -264,7 +264,7 @@ public function isValidUrl( WikiaValidator $validator ) {
 	 */
 	private function getSanitizedData() {
 		$poiData = [
-			'name' => $this->getData( 'name' ),
+			'name' => htmlspecialchars( $this->getData( 'name' ) ),
```

Comment:
Looks like some of these are included in attributes in the templates, so using `ENT_QUOTES` might be a best practice (even though it looks like all attributes are currently using double quotes.